packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -155,15 +155,31 @@ export class Compass {
     const debugClient = debug.extend('webdriver:client');
     const browserProto = Object.getPrototypeOf(browser);
 
-    for (const prop of Object.getOwnPropertyNames(browserProto)) {
+    // We can pull the own property names straight from browser, but brings up a
```

Comment:
I tweaked our addDebugger() method a bit to also add tracing for webdriverios' builtin commands. Previously we only added debug logging to our own custom commands. If you have a long-ish run of calls to webdriver methods rather than our custom ones then the debug output can be super confusing. Catches me all the time.

Hopefully this helps. Some logging inside a whileUntil loop might be a bit much, but that's usually just for the failing tests and the long run of identical logs might actually make it more obvious to see what went on.